B827EBFFFE97807A.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "b827ebfffe97807a", 
+		"servers": [ 
+			{ 
+				"server_address": “us-west.thethings.network“, 
```

Comment:
The character used for double quotes is invalid, please use standard double quote.